spec/features/edit_me_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe "User can manage their own profile" do
```

Comment:
I hear you. I thought about to do that, but profile editing always acts upon the `current_user`. 

How would I test that `current_user` could edit the profile of someone else?